test/check-machines-snapshots

```diff
@@ -469,6 +482,101 @@ class TestMachinesSnapshots(machineslib.VirtualMachinesCase):
         self.assertEqual("no", m.execute("virsh snapshot-info --domain subVmTest1 --snapshotname snapshotB | grep 'Current:' | awk '{print $2}'").strip())
         self.assertEqual("yes", m.execute("virsh snapshot-info --domain subVmTest1 --snapshotname snapshotC | grep 'Current:' | awk '{print $2}'").strip())
 
+    def testMemoryLocation(self):
```

Comment:
It would be good to test this for system and user. The latter has a different hardcoded default, there's permissions and SELinux involved and such. I'm ok with that being a follow-up, as we don't test current snapshots for users either.